mindsdb/api/mysql/mysql_proxy/executor/executor_commands.py

```diff
@@ -682,10 +682,26 @@ def answer_create_chatbot(self, statement):
         # Database ID cannot be null
         database_id = database['id'] if database is not None else -1
 
+        # To be added by https://github.com/mindsdb/mindsdb_sql/pull/307.
+        # TODO: change to statement.agent when possible.
+        agent_node = getattr(statement, 'agent', None)
+        agent = None
+        if agent_node is not None and agent_node.get_string() != 'NULL':
```

Comment:
I think we can just left 'if agent_node is not None', without checking for !='NULL'
And  model_node too